cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -29,16 +29,18 @@ func (suite *InstallerIntegrationTestSuite) TestInstallFromLocalSource() {
 	ts.UseDistinctStateExes()
 	target := filepath.Join(ts.Dirs.Work, "installation")
 
+	sourceDir := suite.populateSourceDir(ts)
+
 	// Run installer with source-path flag (ie. install from this local path)
 	cp := ts.SpawnCmdWithOpts(
 		ts.InstallerExe,
-		e2e.WithArgs(target, "--source-path", filepath.Dir(ts.InstallerExe)),
+		e2e.WithArgs(target, "--source-path", sourceDir),
```

Comment:
Can you summarize why this was necessary again? I wonder if the problem here was that `ts.InstallerExe` was pointing to the wrong file to begin with.

Reply:
When I initially ran these tests I kept getting `Installing Update` not `Installing State Tool`. This was because the `state-installer` binary was being run with the state tool next to it. When we made those updates we may not have updated this test. The `sourceDir` contains only our 3 executables.

Reply:
I see. I think maybe a cleaner solution then is to put the installer in a subdir:

http://github.com/ActiveState/cli/blob/950bf1083327954d4a1e807c14de37fd26d771f7/internal/testhelpers/e2e/session.go#L138-L138